### PR TITLE
fix: added link for screen shots of entire app

### LINKED ISSUE #661: fix: Add screen shots of the entire app in Readme.md file
**Summary:**
The Readme of the app still misses a lot of screen shots of the app. 
We can show many of our fragments.
I want to add the rest of the pages in the screenshot section of Readme

### PR DESCRIPTION
**Pull request by Bhavnaharitsa**
Fixes: #661 
Added link for the entire screen shot of the app.

- [.] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [.] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [.] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Bhavnaharitsa** [comment]
@shiv07tiwari Please review

**Comment by Bhavnaharitsa** [comment]
@luckyman20 Please review

**Comment by Bhavnaharitsa** [comment]
@luckyman20 Please review this pr...I have added the Screenshots file and linked it here. If we merge this, the task will be completed

**Comment by Bhavnaharitsa** [comment]
@luckyman20 Please merge this sir :)

**Comment by Bhavnaharitsa** [comment]
@luckyman20 Please review this

**Review by luckyman20 — COMMENTED** [review]
Please add descriptions to the screenshots and arrange them nicely.

**Comment by Bhavnaharitsa** [comment]
@luckyman20 This pr only adds the link of the file. The arrangement stuff I have done in the previous pr where I have added the Screenshots. This pr just gives the link to it.

**Comment by Bhavnaharitsa** [comment]
> Please add descriptions to the screenshots and arrange them nicely.

But if you want me to add description, I can open it as a new pr.

**Comment by luckyman20** [comment]
@Bhavnaharitsa Just do it in this PR and update the commit description.

**Comment by Bhavnaharitsa** [comment]
Shall do it :)

> @Bhavnaharitsa Just do it in this PR and update the commit description.

**Comment by rachittt** [comment]
Hello @Bhavnaharitsa, instead of removing the current images we can simply write "Click here for more images" under the existing images. So that there is a preview sort of thing for the viewer and it also allows him/her to check all the screenshots.

**Comment by Bhavnaharitsa** [comment]
@rachittt I dont think theres a need to show the older screen shots The newer screenshots are already updated and this pr like you said, is just a link. The discussion was to add a small description under it and then post a link :)

**Comment by rachittt** [comment]
@Bhavnaharitsa we have to update the screenshots in the README to the new ones and then write "Click here for more screenshots" under those screenshots. We don't want to remove all the images.

**Comment by Bhavnaharitsa** [comment]
> @Bhavnaharitsa we have to update the screenshots in the README to the new ones and then write "Click here for more screenshots" under those screenshots. We don't want to remove all the images.

Okay I got it. "Click here for more images" will have the older screen shots? Please help me clarify:)

**Comment by rachittt** [comment]
> more

"Click here for more screenshots" will redirect to [Screenshots.md](https://github.com/openMF/mobile-wallet/blob/dev/Screenshot.md)

**Comment by Bhavnaharitsa** [comment]
> > more
> 
> "Click here for more screenshots" will redirect to [Screenshots.md](https://github.com/openMF/mobile-wallet/blob/dev/Screenshot.md)

Okay!

File: README.md [hunk 1]
```diff
@@ -32,7 +27,7 @@ We have the following branches :
  * **dev**
      All the contributions should be pushed to this branch. If you're making a contribution,
      you are supposed to make a pull request to _dev_.
-     Please make sure it passes a build check on Travis.
+     Please make sure it passes a build check on Travi
```
Comment (shiv07tiwari) [inline README.md:30]: @Bhavnaharitsa Please fix this.

Comment (shiv07tiwari) [inline README.md:30]: @Bhavnaharitsa Please fix this.

**Comment by shiv07tiwari** [comment]
@Bhavnaharitsa Any updates on this?

**Comment by rchtgpt** [comment]
This PR comes from an unknown repository, hence no further changes can be made to it.

**Pull request closed without merging**